### ISSUE TITLE
Mary-Kate and Ashley Sweet 16 - invalid arm condition

### ISSUE THREAD

**Issue by xTibor** [michelhe/rustboyadvance-ng#105]
Smashing the A button in the menu should trigger the panic.

```
thread 'main' panicked at 'invalid arm condition', rustboyadvance-core/src/arm7tdmi/cpu.rs:403:21
```

**Comment by michelhe** [comment]
Solved with open-bus #54

**Comment by michelhe** [comment]
Fixed in c8d5094
